docs/recipes/publishing_notebooks_into_the_DLC_main_cookbook.md

```diff
@@ -0,0 +1,78 @@
+# Publishing Notebooks into the Main DLC Cookbook
+*Date: 13 June 2023*
+
+
+## Introduction
```

Comment:
In general I would add information to first check whether a recipe or tutorial already exists and can be expanded. For instance, if it's related to I/O, maybe it can be added to I/O manipulations instead of being a completely new thing. 
Definitely information about the structure of a recipe. We should expect that there will be some intro explaining why this can be useful, followed by examples/workflow.

Reply:
Great inputl, thanks @KonradDanielewski!

Reply:
Hi @KonradDanielewski ! I pushed new changes incorporating your comments. They were really helpful in improving the recipe. Thank you so much! 🎉